src/common/utility_win.cpp

```diff
@@ -147,8 +148,8 @@ void setLaunchOnStartup_private(const QString &appName, const QString &guiName,
     }
 }
 
-// TODO: Right now only detection on toggle/startup, not when windows theme is switched while nextcloud is running
-static inline bool hasDarkSystray_private()
+#ifndef TOKEN_AUTH_ONLY
```

Comment:
I think you should remove this
if `TOKEN_AUTH_ONLY` is defined, the build would most probably fail ?